lib/lokka/app.rb

```diff
@@ -28,24 +26,34 @@ class App < Sinatra::Base
       set :admin_per_page, 200
       set :default_locale, 'en'
       set :haml, attr_wrapper: '"'
+      set :session_secret, 'development' if development?
       set :protect_from_csrf, true
       supported_stylesheet_templates.each do |style|
         set style, style: :expanded
       end
       ::I18n.load_path += Dir["#{root}/i18n/*.yml"]
-      helpers Lokka::Helpers
-      helpers Lokka::RenderHelper
-      use Rack::Session::Cookie,
```

Comment:
As `enable :sessions` are configured, we don't need to explicitly `use Rack::Session::Cookie`.